Rakefile

```diff
@@ -51,7 +51,7 @@ end
 def defaults(args)
   bindir = args.bindir.nil? || args.bindir.empty? ? Pathname.new('./build/swiftgen/bin') : Pathname.new(args.bindir)
   fmkdir = args.fmkdir.nil? || args.fmkdir.empty? ? bindir + '../lib' : Pathname.new(args.fmkdir)
-  tpldir = args.tpldir.nil? || args.tpldir.empty? ? bindir + '../templates' : Pathname.new(args.tpldir)
+  tpldir = args.tpldir.nil? || args.tpldir.empty? ? bindir + '../Resources/templates' : Pathname.new(args.tpldir)
```

Comment:
@AliSoftware Great question, my thinking was it'd be best to keep as much parity as makes sense between the debug and the packaged version. Based on the rake install step I assumed that adding the default value would be the lowest touch way to keep things harmonious (if the debug version requires default path `Resources/templates` running install will overwrite the `main.swift` file and require reset). However I certainly understand that the `.app` is a secondary feature. Some other options I considered were:
- A build script to package the .app more like the cli tool (discussed [here](https://github.com/AliSoftware/SwiftGen/issues/169))
- Using a different relative path value depending on the target.
- Resetting the path value in `main.swift` after the install step finishes in the rake file.

Let me know if any of these options appeal to you and I'd be happy to implement it (or if I'm off in the wrong direction all together 😬)